xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-ui/src/main/resources/XWiki/OfficeImporterAdmin.xml

```diff
@@ -91,10 +91,8 @@
       ### Info about the place to modify the configuration
       ###
       &lt;p&gt;
-        &lt;span class="box infomessage"&gt;
-          ## TODO: we do not escape this string because we inject some HTML code. We should improve it.
-          $services.localization.render('admin.ooserver.options.source', ['&lt;span class="monospace"&gt;xwiki.properties&lt;/span&gt;'])
-        &lt;/span&gt;
+        ## TODO: we do not escape this string because we inject some HTML code. We should improve it.
```

Comment:
@michitux might be good to have your input on this...

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3555/commits/4b60a9234c155ecc698d5c50025779dfef58cc9d :+1: 
Here is what this looks like on my local instance
![image](https://github.com/user-attachments/assets/f4be1107-a998-4c7a-acb0-4297cd3842ee)
$${\color{lightgreen}ADDRESSED}$$